pom.xml

```diff
@@ -166,7 +174,7 @@
     <javax-inject.version>1</javax-inject.version>
     <log4j.version>2.13.2</log4j.version>
     <netty.version>4.1.51.Final</netty.version>
-    <swirlds.version>0.15.1</swirlds.version>
+    <swirlds.version>0.16.0-alpha.1</swirlds.version>
```

Comment:
Will merge this platform SDK to unblock HIP-17 work for now. Created new issue #1631 to use `FCOneToManyRelation` data structure in TokenStore.